examples/dockerfile2llb/main.go

```diff
@@ -31,6 +32,7 @@ func xmain() error {
 	var opt buildOpt
 	flag.StringVar(&opt.target, "target", "", "target stage")
 	flag.StringVar(&opt.partialImageConfigFile, "partial-image-config-file", "", "Output partial image config as a JSON file")
+	flag.StringVar(&opt.sourceImageConfigFile, "source-image-config-file", "", "Output source image config as a JSON file")
```

Comment:
I think it should be called base image, not source image here and everywhere else.

Reply:
Isn't "base image" confusing in this case?

```
FROM busybox AS foo

# the "base image" of bar is busybox, not foo
FROM foo AS bar
```

Reply:
But isn't the point of this feature to keep the timestamps from "busybox" in this case? So yes it is the base image.

In here I think it makes sense to look at the base image based on the layers from the result artifact, not as a structure of Dockerfile stages. This is also how it works in provenance when we export layers or how Docker Scout detects if the base image should be updated (as opposed to source materials that is an array and not a single image).

Reply:
Renamed to "base" images

> In here I think it makes sense to look at the base image based on the layers from the result artifact, not as a structure of Dockerfile stages. 

I'm not sure about this. Can this be discussed in a separate PR?